main/perl-math-basecnv/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Contributor: Natanael Copa <ncopa@alpinelinux.org>
+# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
+pkgname=perl-math-basecnv
+pkgver=1.14
+pkgrel=0
+pkgdesc="Math::BaseCnv perl module"
+url="http://search.cpan.org/dist/Math-BaseCnv/"
+arch="noarch"
+license="GPL-3.0"
+depends="perl"
+makedepends="perl-dev"
+subpackages="$pkgname-doc"
+source="http://search.cpan.org/CPAN/authors/id/P/PI/PIP/Math-BaseCnv-$pkgver.tgz"
+options="!check"
+
+builddir="$srcdir"/Math-BaseCnv-$pkgver
+
+build() {
+	cd "$builddir"
+	PERL_MM_USE_DEFAULT=1 perl Makefile.PL INSTALLDIRS=vendor
+	make && make test || return 1
```

Comment:
`make test` can be moved to a `check()` function. Then you can remove `options="!check"`

`|| return 1` is no longer necessary